Modules/timemodule.c

```diff
@@ -765,7 +765,7 @@ time_strftime(PyObject *self, PyObject *args)
         return NULL;
     }
 
-#if defined(_MSC_VER) || (defined(__sun) && defined(__SVR4)) || defined(_AIX)
+#if defined(_MSC_VER) || (defined(__sun) && defined(__SVR4)) || defined(_AIX) || defined(__VXWORKS__)
```

Comment:
VxWorks libc can't handle years out of range [1; 9999]. All years less than 1 will be treated as 0 by the libc. And the most year digit VxWorks can handle is 4. That is, any year that has more than 4 digits will be treated as 4 digit year.